applications/StructuralMechanicsApplication/custom_strategies/custom_schemes/explicit_central_differences_scheme.hpp

```diff
@@ -479,12 +538,9 @@ class ExplicitCentralDifferencesScheme
         for (IndexType j = 0; j < DomainSize; j++) {
             if (fix_displacements[j]) {
                 r_current_acceleration[j] = 0.0;
-                r_middle_velocity[j] = 0.0;
+                r_current_velocity[j] = 0.0;
```

Comment:
this is not correct: if the displacement is fixed to a given value, different from zero than the velocity and acceleration are not zero

Reply:
yes that is true, thanks

Reply:
> > Ok, I was having problems with the falling sphere test written by you @loumalouomega. One can actually also see in this test that the current strategy lags 1 time step for velocity and acceleration as the velocity in the 1st time step currently is 0 in this case which is wrong.
> > There are actually more problems:
> > ```
> > 1. As we first have to update the displacement and middle_vel before calculating the element residuals, we somehow should also call MoveMesh and all the finalize operations within the time step. I think this is not feasible.
> > 
> > 2. I need to call `ConstraintUtilities::PreComputeExplicitConstraintConstribution` in the scheme now right after the element residuals have been updated. Because I cannot split `Update`.
> > ```
> > 
> > 
> > I'd be happy to skype, also including @philbucher
> 
> I remember that I "skipped" one step in order to retrieve the analytic solution. That can be updated to the exact solution without additional issues.
> 
> For the other problems:
> 
> 1. It is not necessary to move the mesh...if we update the implementation of the elements...
> 2. I do not know how to solve this issue...

The mesh should be moved ... the point is that imagine that you want to solve a thermal problem on the top of it.
The thermal solver is not aware of the mesh motion, it simply uses the deformed configuration

The only thing that elements need to take into account is that if the element is linear they should operate over the original configuration instead on the moved one